### PR TITLE
Update: Add missing aria attribution for tabs

### LINKED ISSUE #71: adapt-contrib-resources Missing tab structure for resources tabs
## Bug Description:
Tabs in the Resources dialog do not have tab structure to convey their association or expected behavior (for example, which tab is active within the grouping).

## Steps to Reproduce:

    Using a screen reader, activate the resources button.
    Activate the Resources option, then navigate through the [All/Document/Media/Link] tabs and note their role and state. 

## Actual Behavior:

All tab elements are announced as standalone buttons by screen readers. Their is no indication that these elements are related, or of which tab in the group is selected/active.

## Expected Behavior:

    Ensure tabs are contained in an element with role="tablist"
    Ensure tabs are structured with role="tab"
    Ensure the selected tab has the aria-selected property applied
    Ensure the panel controlled by the tabs is structured with role="tabpanel" 

Ensure the keyboard behavior of the tablist is consistent with behavior defined in the ARIA Authoring Practices

Refer to the ARIA Authoring practices for guidance on compliant tab structure: [​https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel](https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel)

### PR DESCRIPTION
**Pull request by chris-steele**
fixes #71 (see https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/)

### CODE REVIEW

**Comment by oliverfoster** [comment]
I'm always weary of adding keyboard controls.

1. The specs are rarely uniform. https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/ https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-automatic.html https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role https://w3c.github.io/aria-practices/#tabpanel
2. Multi-device keyboard testing is required for the keyboard controls to ascertain the compatibility with each device.
3. The behaviour should be uniform across all plugins of the same role type, such that the code will be duplicated across many plugins.


Possible solutions:
1. Use the minimum required controls.
2. Make sure at least iOS + Voiceover, Chrome and Edge and Firefox all work with both NVDA and Jaws.
3. Look at externalising the keyboard controls into the core and attaching them using an API.

**Review by joe-replin — COMMENTED** [review]
Good changes, all! The tablist, tabs & tabpanel are all working according to spec and the aria-labelledby attribute is dynamically referencing the selected tab. JAWS & NVDA testing all performed well in Windows - Chrome, Firefox & Edge. 

macOS - Voiceover - Chrome & Firefox worked well but the arrow keys weren't toggling through the tabs like it does on my Windows machine.

**Comment by chris-steele** [comment]
@joe-replin @oliverfoster given the immediate issues that have been found and the increased weight on the test matrix I suggest we drop the keyboard controls from this PR, would you agree?

**Comment by joe-replin** [comment]
I believe the keyboard controls were flagged as a 'minor' issue within in the feedback. I agree with @oliverfoster's point with externalizing the keyboard controls and accessing them whenever we need so we aren't copying the same code into individual plug-ins whenever we need.

**Comment by chris-steele** [comment]
Removed key event handlers - we will need to look at this externalisation in another plugin and do the same for [adapt-contrib-matching](https://github.com/adaptlearning/adapt-contrib-matching/) and [adapt-contrib-slider](https://github.com/adaptlearning/adapt-contrib-slider/)

**Review by joe-replin — APPROVED** [review]
Looks great, @chris-steele. Thank you!

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-resources",
-  "version": "5.0.1",
+  "version": "5.0.2",
```
Comment (oliverfoster) [inline bower.json:3]: version numbers are always bumped by releaser, please revert in the readme and bower.json

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by guywillis — APPROVED** [review]
👍  Tested with NVDA 2020.1 in Chrome

**Pull request merged**